prisma-fmt-wasm/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@prisma/prisma-fmt-wasm",
+  "version": null,
+  "description": "",
+  "main": "src/prisma_fmt_build.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/prisma/prisma-engines"
```

Comment:
```suggestion
    "url": "https://github.com/prisma/prisma-engines",
    "directory": "prisma-fmt-wasm"
```

Reply:
What does this affect? I assume the readme view on npmjs.com?

Reply:
let's do this then